ds-prep-python-dictionaries/main.py

```diff
@@ -0,0 +1,23 @@
+from sys import set_coroutine_origin_tracking_depth
+
+
+person = {"first_name": "Ravleen" , "last_name": "Kaur" , "hobby": "Dancing"}
+print(person)
+print(person["first_name"])
+first_name = person["first_name"]
+print(first_name)
+x = person.get("last_name")
+print(x)
+last_name = person.get("last_name")
+print(last_name)
+x = person.get("middle_name")
+print(x)
+middle_name = person.get("middle_name")
+print(middle_name)
+print(person["first_name"] , ["middle_name"] , ["last_name"])
```

Comment:
There should only be one `print` statement here, and it should print out all three of the `*_name` variables at once:

---

![image](https://user-images.githubusercontent.com/16143771/149248284-4cffa865-0343-4424-af44-2aabb2019bb8.png)

---

![image](https://user-images.githubusercontent.com/16143771/149248203-cedbfa9e-6e03-4999-b151-ab8278b65821.png)